### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JasonFreeberg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,6 +130,7 @@ The above example uses app-level credentials i.e., publish profile file for depl
 Follow the steps to configure the secret:
 
 * Download the publish profile for the WebApp from the portal (Get Publish profile option)
+  * **Note**: As of October 2020, Linux web apps will need the app setting `WEBSITE_WEBDEPLOY_USE_SCM` set to `true`. This requirement will be removed in the future.
```
Comment (ehamai) [inline README.md:133]: The app setting change needs to happen BEFORE downloading the profile because that controls which endpoint they get.

Comment (JasonFreeberg) [reply README.md]: Updated the text

**Comment by aksm-ms** [comment]
@N-Usha please review

File: README.md [hunk 2]
```diff
@@ -129,6 +129,7 @@ The above example uses app-level credentials i.e., publish profile file for depl
 
 Follow the steps to configure the secret:
 
+* **Note**: As of October 2020, Linux web apps will need the app setting `WEBSITE_WEBDEPLOY_USE_SCM` set to `true` before continuing. This requirement will be removed in the future.
```
Comment (N-Usha) [inline README.md:132]: Thanks for adding this @JasonFreeberg. We may also want to add this to individual workflow templates in https://github.com/Azure/actions-workflow-samples/tree/master/AppService for greater visibility of this message.

**Review by N-Usha — APPROVED** [review]

**Pull request merged**